### PR TITLE
refactors simple animal Die() to be death() instead

### PR DESCRIPTION
**Pull request by MadmanMartian**
literally just #14599

Was just ctrl+H Die() -> death(), so will review here for errors

### CODE REVIEW

File: code/modules/mob/living/carbon/alien/special/facehugger.dm [hunk 1]
```diff
@@ -387,7 +387,7 @@
 		GoActive()
 	return
 
-/obj/item/clothing/mask/facehugger/proc/Die()
+/obj/item/clothing/mask/facehugger/proc/death()
```
Comment (MadmanMartian) [inline code/modules/mob/living/carbon/alien/special/facehugger.dm:390]: lmao

File: code/modules/mob/living/simple_animal/hostile/human/surgeon.dm [hunk 2]
```diff
@@ -34,7 +34,7 @@
 
 //	faction = "necro" //don't want him to get killed by skeletons before his transformation
 
-//mob/living/simple_animal/hostile/humanoid/surgeon/boss/Die()
+//mob/living/simple_animal/hostile/humanoid/surgeon/boss/death()
```
Comment (MadmanMartian) [inline code/modules/mob/living/simple_animal/hostile/human/surgeon.dm]: lmao

**Comment by PJB3005** [comment]
You're gonna have a crap load of errors if you just do a find/replace dude. I would've if it was possible. Even when you fix the compile errors there'll be a bunch of runtime bugs due to say the return values.

**Comment by MadmanMartian** [comment]
I'll handle that after lunch then

**Comment by MadmanMartian** [comment]
somewhat handled. Spent a few hours blowing up hitlers, xenos, and mice.

Couldn't produce any noticable bugs. Please get some test jockeys on this.

**Comment by kilozombie** [comment]
People death() when they are killed

**Pull request merged**